app/jobs/results_recorder_job.rb

```diff
@@ -24,9 +24,13 @@ class ResultsRecorderJob < ApplicationJob
   # @param [Integer] version
   # @param [String] s3_part_key
   # @param [String] delivery_class Name of the worker class that performed delivery
-  def perform(druid, version, s3_part_key, _delivery_class)
+  def perform(druid, version, s3_part_key, delivery_class) # rubocop:disable Lint/UnusedMethodArgument used as job.arguments.fourth in before_perform
```

Comment:
It applies to the scope of the block if you do it this way.

Reply:
yeah, i had to double-check whether that was legal, but it seemed nice since the exception is just about the one line, and since i think the rule in general is a good one.  there's one other place in the codebase that does this at the moment (for a class var in `DruidVersionZip`).